teuthology/openstack/setup-openstack.sh

```diff
@@ -670,7 +676,7 @@ function main() {
     : ${nameserver:=$ip}
 
     if $do_create_config ; then
-        create_config "$network" "$subnets" "$nameserver" "$labdomain" "$ip" "$flavor_select" "$archive_upload" || return 1
+        create_config "$network" "$subnets" "$nameserver" "$labdomain" "$ip" "$flavor_select" "$archive_upload" "$canonical_tags" || return 1
```

Comment:
Sending an empty string as a positional argument to a function might not be such a good idea. . . if it's the last argument it should be OK. Right now canonical_tags is the 8th argument. If a ninth argument is added later, it will only be 9th when `canonical_tags != ""` - when canonical_tags is the empty string, the intended 9th argument will be seen as the 8th.